docs/man/failover.txt

```diff
@@ -250,6 +268,11 @@ When using `failover` for redundancy between multiple UPS drivers connected to
 the same underlying UPS device, data is not multiplexed between the drivers. As
 a result, some data points may be available in some drivers but not in others.
 
+For `checkruntime` considerations, the unit of both `battery.runtime` and
+`battery.runtime.low` is assumed to be **seconds**. UPS drivers that report
+these values using different units are considered non-compliant with the NUT
+variable standards and should be reported to the NUT developers as faulty.
+
```

Comment:
General note for posterity: not all NUT drivers provide a `battery.runtime`, some might only have a `battery.charge` (or nothing at all).

For the purpose here, comparing charges is probably rather useless (unless the UPSes have similar capacity so runtimes would happen to compare similarly); but it may be productive to eventually focus on generalizing the `runtimecal` fallback, for these time numbers to be available in all/most drivers => #2420